custom_components/tuya_local/devices/kogan_KASMCDSKTLA_kettle.yaml

```diff
@@ -0,0 +1,65 @@
+name: Kogan 1.7L Double Wall Glass Smart Kettle
+products:
+  - id: 7occ66lr74vfznwt
+    manufacturer: Kogan
+    model: KASMCDSKTLA
+entities:
+  - entity: switch
```

Comment:
I think following the water_heater pattern of other kettles will be better in the long run.